src/main/java/org/jenkinsci/plugins/codesonar/models/HttpServiceResponse.java

```diff
@@ -0,0 +1,47 @@
+package org.jenkinsci.plugins.codesonar.models;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.charset.StandardCharsets;
+
+public class HttpServiceResponse {
```

Comment:
Is `HttpServiceResponse` a "model" class?  Seems like it belongs either in the services or the api subpackages.